apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Tests/Trace/SimpleManagedTracerTest.cs

```diff
@@ -56,14 +58,14 @@ private static bool IsValidSpan(TraceSpan span, string name, ulong parentId, Spa
         [Fact]
         public void SingleSpan()
         {
-            Mock<ITraceConsumer> mockConsumer = new Mock<ITraceConsumer>();
+            Mock<IConsumer<TraceProto>> mockConsumer = new Mock<IConsumer<TraceProto>>();
             IManagedTracer tracer = SimpleManagedTracer.Create(mockConsumer.Object, CreateTrace());
 
             mockConsumer.Setup(c => c.Receive(
-                Match.Create<Traces>(
-                    t => t.Traces_.Count == 1 &&
-                        t.Traces_[0].Spans.Count == 1 &&
-                        IsValidSpan(t.Traces_[0].Spans[0], "span-name"))));
+                Match.Create<IEnumerable<TraceProto>>(
+                    t => t.Count() == 1 &&
+                        t.ElementAt(0).Spans.Count == 1 &&
```

Comment:
How about `t => IsValidSpan(t.Single().Spans.Single(), "span-name"))`

That validates the "single-ness" of the trace and the span. (Ditto below.)